executor/show_stats_test.go

```diff
@@ -231,7 +232,7 @@ func (s *testShowStatsSuite) TestShowAnalyzeStatus(c *C) {
 	c.Assert(result.Rows()[0][0], Equals, "test")
 	c.Assert(result.Rows()[0][1], Equals, "t")
 	c.Assert(result.Rows()[0][2], Equals, "")
-	c.Assert(result.Rows()[0][3], Equals, "analyze table")
+	c.Assert(strings.Contains(result.Rows()[0][3].(string), "analyze table"), IsTrue)
```

Comment:
Would it be unstable if we check the full text?

Reply:
Because I added other informations after analyze table. Prefer Contains to assert substring needed instead of full text.